encoders/src/main/scala/sql/derivation/Deserializer.scala

```diff
@@ -0,0 +1,62 @@
+package sql.derivation
+
+import scala.compiletime
+import scala.deriving.Mirror
+import scala.reflect.ClassTag
+
+import org.apache.spark.sql.catalyst.expressions.{Expression, If, IsNull, Literal}
+import org.apache.spark.sql.catalyst.expressions.objects.NewInstance
+import org.apache.spark.sql.catalyst.analysis.UnresolvedExtractValue
+import org.apache.spark.sql.catalyst.DeserializerBuildHelper._
+import org.apache.spark.sql.types._
```

Comment:
RC2 around is almost here!
```suggestion
import org.apache.spark.sql.catalyst.DeserializerBuildHelper.*
import org.apache.spark.sql.types.*
```